### DISCUSSION TITLE
Cannot Login to Codemagic account

### DISCUSSION THREAD

**Discussion by FozanKh** [codemagic-ci-cd/codemagic-docs#2347] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1- Using Brave Browser, Navigate to https://codemagic.io/login
2- Type my email
3- Click on login
4- it says "We have sent you an email to xxx@xxx.com with a six-character authentication key. Enter the key you received and click Continue to finalize your login. The key is case-sensitive.", however nothing is received.

### Expected results

I would received auth key to my email, enter it and login

### Actual results

No auth key was sent to my email

**Comment by dtrdic** [comment, marked as answer]
Hello @FozanKh is the email containing the authentication key not arriving at all, or is there a delay in its delivery? Please check your spam/junk folders as well.  If there is a delay, it might be caused by spam/virus filters on your end. If you have implemented any spam filters, consider whitelisting our domain [no-reply@codemagic.io](mailto:no-reply@codemagic.io)

**Reply by FozanKh** [reply]
Thank for your help, issue is solved!
